test/hotspot/jtreg/compiler/vectorapi/VectorGatherScatterTest.java

```diff
@@ -42,7 +38,6 @@
  * @bug 8288397
  * @key randomness
  * @library /test/lib /
- * @requires vm.cpu.features ~= ".*sve.*" & (vm.opt.MaxVectorSize == "null" | vm.opt.MaxVectorSize >= 16)
```

Comment:
We have to make sure this test is run with "MaxVectorSize >= 16", or following code will throw exception due to no vector shape maps (e.g. `32-bit/8-byte`):
```
 private static final VectorSpecies<Float> F_SPECIES =
            VectorSpecies.of(float.class, VectorShape.forBitSize(L_SPECIES.vectorBitSize() / 2));
```
We added the this requirement in case people run the tests with "-XX:MaxVectorSize=8".

Reply:
Got it let me add that back